eas.json

```diff
@@ -57,7 +79,10 @@
       },
       "env": {
         "APP_ENV": "development",
-        "EXPO_NO_DOTENV": "1"
+        "EXPO_NO_DOTENV": "1",
+        "API_URL": "https://dummy.com",
+        "VAR_NUMBER": "1",
+        "VAR_BOOL": "true"
```

Comment:
Correct me if i'm wrong @asdolo, these ones are not necessary anymore with your changes right?

Reply:
That's right! They're not necessary anymore 🙂